core-bundle/src/EventListener/CacheListener.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\EventListener;
+
+use Symfony\Component\HttpKernel\Event\FilterResponseEvent;
+
+class CacheListener
```

Comment:
This class name does not really explain what the listener does…

Reply:
Proposals?

Reply:
`MakeResponsePrivateListener`?

Reply:
Renamed in https://github.com/contao/contao/pull/389/commits/2adefcaef9cfa6cccb52edaf2d3e5f2714ddf022.

Reply:
I find an "action naming" strage, I would have chosen `PrivateResponseListener` if that's what it's about. Or just `PrivateCacheListener`. However, is the response actually "private" or is it "no-cache" if there e.g. is a session? So maybe rather `UncacheableResponseListener`?

Reply:
The response becomes private, which is correct. It may still be cached by the client, we don't care. So `UncacheableResponseListener` would be wrong. And it makes the response private but the listener itself is not so `PrivateCacheListener` sounds wrong too. It sounds like it's a listener that is only there for the private cache but it's exactly the opposite 😄 It's there to ensure that certain responses are never cached by any reverse proxy even if the developer said so.